Dockerfile.kernel

```diff
@@ -1,25 +1,18 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-#FROM jupyter/minimal-kernel
 FROM jupyter/minimal-kernel:9cb6443840b2
```

Comment:
It's always good to pin to a version regardless. docker-stacks change a lot and sometimes in backward incompatible ways.